SDL/Makefile

```diff
@@ -72,16 +94,24 @@ CUTVALUEFLAG_FLAGS = -DCUT_VALUE_DEBUG
 %_cuda.o: %.cc
 	$(COMPILE_CMD_CUDA) $(CXXFLAGS_CUDA) $(ROOTINCLUDE) $(PRINTFLAG) $(CACHEFLAG) $(CUTVALUEFLAG) $(LSTWARNINGSFLAG) $(CMSSW_WERRORS_CUDA) $(T5CUTFLAGS) $(NOPLSDUPCLEANFLAG) $(TCPLSTRIPLETSFLAG) $(PTCUTFLAG) $(DUPLICATES) $(ALPAKAINCLUDE) $(ALPAKABACKEND_CUDA) $< -o $@
 
-$(LIB_CUDA): $(CCOBJECTS_CUDA) $(LSTOBJECTS_CUDA)
-	$(LD_CUDA) $(SOFLAGS_CUDA) $^ -o $@
-	mkdir -p cuda
-	ln -sf ../$@ cuda/$(@:_cuda.so=.so)
+%_rocm.o: %.cc
+	$(COMPILE_CMD_ROCM) $(CXXFLAGS_ROCM) $(ROOTINCLUDE) $(PRINTFLAG) $(CACHEFLAG) $(CUTVALUEFLAG) $(LSTWARNINGSFLAG) $(CMSSW_WERRORS_ROCM) $(T5CUTFLAGS) $(NOPLSDUPCLEANFLAG) $(TCPLSTRIPLETSFLAG) $(PTCUTFLAG) $(DUPLICATES) $(ALPAKAINCLUDE) $(ALPAKABACKEND_ROCM) $< -o $@
 
 $(LIB_CPU): $(CCOBJECTS_CPU) $(LSTOBJECTS_CPU)
 	$(LD_CPU) $(SOFLAGS_CPU) $^ -o $@
 	mkdir -p cpu
 	ln -sf ../$@ cpu/$(@:_cpu.so=.so)
 
+$(LIB_CUDA): $(CCOBJECTS_CUDA) $(LSTOBJECTS_CUDA)
+	$(LD_CUDA) $(SOFLAGS_CUDA) $^ -o $@
+	mkdir -p cuda
+	ln -sf ../$@ cuda/$(@:_cuda.so=.so)
```

Comment:
do we really need these softlinks now? IIUC, the dependencies are built depending on a specific `libsdl_xyz` and the LD_LIBRARY_PATH substitution is not used anymore

Reply:
That's true. Those are not needed after your PR. I'll remove them.